src/main/kotlin/com/openlattice/organizations/processors/UpdateOrganizationExternalDatabaseTable.kt

```diff
@@ -0,0 +1,48 @@
+package com.openlattice.organizations.processors
+
+import com.kryptnostic.rhizome.hazelcast.processors.AbstractRhizomeEntryProcessor
+import com.openlattice.edm.requests.MetadataUpdate
+import com.openlattice.organization.OrganizationExternalDatabaseTable
+import java.util.*
+
+class UpdateOrganizationExternalDatabaseTable(val update: MetadataUpdate) :
```

Comment:
You can make it into a data class, so you don't need to implement `equals `, `hashCode` and `toString`